### PR TITLE
feat: move credential docs into concept section

### PR DESCRIPTION
**Pull request by tjwelde**
## fixes KILTProtocol/ticket#1969
This PR moves the credential docs into the concept section, revising them in the process and adding an overview page.

### CODE REVIEW

**Comment by tjwelde** [comment]
Not sure about writing things like Claimer, Attester, Attestation, Claim, etc always capitalized, or even in code blocks, like `Request for Attestation`...

File: docs/concepts/credentials/0_overview.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+**Credentials** are attested claims which are owned by a **Claimer**, attested by an **Attester** and can be verified by a **Verifier**.
+
+To get a Credential a Claimer needs to go through following process:
+1. Find a **CType**, a Claim should be based on (Can be advertised by the potential Attester and Verifier)
+2. Make a **Claim**
+3. Potentially request and receive **Terms** and agree on a **Quote** with an Attester
+4. **Request an Attestation**
+5. Wait for the Claim to be **attested** on-chain by the Attester
```
Comment (ntn-x2) [inline docs/concepts/credentials/0_overview.md]: Would not put too many details in here.

```suggestion
5. Wait for the Claim to be **attested** by the Attester
```

File: docs/concepts/credentials/0_overview.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+To verify the Credential a Claimer can send a presentation of it to a Verifier
+
+1. The Verifier may request a **Credential** as the first step, providing a nonce for security reasons
```
Comment (ntn-x2) [inline docs/concepts/credentials/0_overview.md]: Same here, probably does not add anything valuable considering this is an introductory section, does it?

```suggestion
1. The Verifier may request a **Credential** as the first step
```

File: docs/concepts/credentials/0_overview.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+2. The Claimer selects properties he wants to **disclose** and signs the Presentation of the Credential including the nonce
```
Comment (ntn-x2) [inline docs/concepts/credentials/0_overview.md]:
```suggestion
2. The Claimer selects properties they want to **disclose** and signs the Presentation of the Credential
```

File: docs/concepts/credentials/0_overview.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+3. The Claimer sends the **Presentation** to the Verifier
+4. The Verifier **checks** the signature, nonce, claim contents and decides, if he trusts the Attester
```
Comment (ntn-x2) [inline docs/concepts/credentials/0_overview.md]:
```suggestion
4. The Verifier **checks** its structure and signature and decides, if they trust the Attester
```

File: docs/concepts/credentials/1_ctypes.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ When making a claim for a CType, all the following properties are required:
   - Time format e.g. T18:25:43.511Z
   - URI format e.g. https://www.example.com
 
-```js
+```js title="CType schema example"
```
Comment (ntn-x2) [inline docs/concepts/credentials/1_ctypes.md]: What happened in the workshop and getting started in the past, is that these snippets become outdated. Is there a way to test also these simple JSON objects, perhaps by putting them into a script run by the CI and then simply embedded here, like we do for code snippets?

File: docs/concepts/credentials/1_ctypes.md [hunk 6]
```diff
@@ -39,6 +39,21 @@ When making a claim for a CType, all the following properties are required:
...
+{
+  schema:  {...} // as defined in the example above>,
+  owner: null, // DID of the owner, or null
+  hash: '0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010' // For looking up on-chain
+}
+```
+
+## Nested CTypes
+
+A Nested CType is a hierarchical composite schema that includes other simpler CTypes as substructures. The Nested CType references the included sub-CTypes. For example, a company could use a Nested CType that includes the required credentials, qualifications, health and safety certs, etc of its current employees.
```
Comment (ntn-x2) [inline docs/concepts/credentials/1_ctypes.md]: Please split up each sentence on a new line, so it is easier to comment in review.

Comment (ntn-x2) [inline docs/concepts/credentials/1_ctypes.md]: Do we want to capitalise also Nested?

```suggestion
A Nested CType is a hierarchical composite schema that includes and references other simpler CTypes as substructures. For example, a company could use a Nested CType that includes the required credentials, qualifications, health and safety certs, etc of its current employees.
```

File: docs/concepts/credentials/1_ctypes.md [hunk 7]
```diff
@@ -84,23 +99,45 @@ nested = {
...
+      name: {
+        title: { default: "Full Name" },
+        description: {
+          default: "The full name of the license holder"
+        }
+      },
+      age: {
+        title: { default: "Age" }
+        description: {
+          default: "The age og the license holder"
```
Comment (ntn-x2) [inline docs/concepts/credentials/1_ctypes.md]:
```suggestion
          default: "The age of the license holder"
```

File: docs/concepts/credentials/1_ctypes.md [hunk 8]
```diff
@@ -84,23 +99,45 @@ nested = {
...
-  - Description
+```js
+{
+  ctypeHash: "0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010",
+  metadata: {
+    title: {
+      default: "Drivers License"
+    },
+    description: {
+      default: "A demo CType for Driver Licenses"
```
Comment (ntn-x2) [inline docs/concepts/credentials/1_ctypes.md]:
```suggestion
      default: "A demo CType for driver's licenses"
```

File: docs/concepts/credentials/2_claiming.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
+---
+id: claiming
+title: Claiming
+---
+There are three actors in the KILT workflow: Claimers, Attesters and Verifiers.
```
Comment (ntn-x2) [inline docs/concepts/credentials/2_claiming.md]: We are already at the second page, should we maybe put this somewhere earlier in the section?

File: docs/concepts/credentials/2_claiming.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+As KILT is an open system: any entity can make a claim about anything. Therefore, as in the real world, a claim only has value if the verifier trusts the attester.
```
Comment (ntn-x2) [inline docs/concepts/credentials/2_claiming.md]:
```suggestion
As KILT is an open system, any entity can make a claim about anything. Therefore, as in the real world, a claim only has value if the verifier trusts the attester.
```

File: docs/concepts/credentials/2_claiming.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+In KILT, claims are based on claim types (CTypes). A claimer can either create a new CType or, when KILT is established and standard CTypes are available, may use an existing CType when creating their claim.
+
+## CTypes
```
Comment (ntn-x2) [inline docs/concepts/credentials/2_claiming.md]: Why are we re-presenting CTypes again? Wasn't the previous section enough?

File: docs/concepts/credentials/2_claiming.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+  contents: { name: 'Alice', age: 29 },
+  owner: 'did:kilt:light:004rVETkZQcK9aBr6SHZXaHQSDyqFFMW2rN5HtEooWgdB92JMg'
+}
+```
+
+## Request an Attestation
+
+After creating your claim, you can request an attestation by creating another object called `RequestForAttestation`, which you can send to the attester of your choice via any messaging system.
+
+We recommend encrypting this object before sending it by calling encrypt on the message object. Additionally, you can also compress the body of the message.
```
Comment (ntn-x2) [inline docs/concepts/credentials/2_claiming.md]: I would say this is an advanced concept not to be put in the overview section.

```suggestion
We recommend encrypting this object before sending it by calling encrypt on the message object.
```

File: docs/concepts/credentials/2_claiming.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+    '0xa919076ce52e7a5e3d90fb3b8b7d0f8931cbfb3c8ab92d25369bb977948e6b71': 'a73eeb42-21bb-474c-ba76-51006d5c8f64',
+    '0x9848c59c5ec1e05bf60e49736623ee1eb531496a53f04322d9442cf49d96333c': '9ce4a717-9896-4d1c-96e3-20499910dba7'
+  },
+  legitimations: [],
+  delegationId: null,
+  rootHash: '0x9f1d98c9cbe7bcfa8e46b314ecb62eac7c42f7cbbeac0dea17edfce3ee68dee9',
+  claimerSignature: {
+    signature: '0x44b8fe2210e1ca29bbdfaadc469c968369a35537a984f223f0806c2b27fb666641de4154649ed47732556d17862304ee9d27a74453d703097506d73c70d7dd8a',
+    keyId: 'did:kilt:light:004rVETkZQcK9aBr6SHZXaHQSDyqFFMW2rN5HtEooWgdB92JMg#authentication',
+    challenge: undefined
```
Comment (ntn-x2) [inline docs/concepts/credentials/2_claiming.md]: Can't this property just be omitted?

File: docs/concepts/credentials/3_terms-and-quote.md [hunk 14]
```diff
@@ -3,36 +3,31 @@ id: requesting-terms-and-attestations
...
+- An attester creates an `Terms` object and sends it as part of a submit terms message back to the claimer.
+- The claimer receives the terms message, checks the terms and, if all is in order, agrees to them
 
 ## How to create a terms object
 
-When creating a terms object, at least one of the following must be complete: Legitimations, Quote or Delegations.
+When creating a terms object, at least one of the following must be set: Legitimations, Quote or Delegations.
 
-- **Legitimations**: Legitimations are composed of attested claims that the attester has issued previously, showing that the attester has the authority or legitimacy to attest the claim requested. This is a way of establishing trust between the participants.
+- **Legitimations**: Legitimations are Credentials, issued to the attester the claimer wants to interact with, showing that the attester has the authority or legitimacy to attest the claim requested. This is a way of establishing trust between the participants.
```
Comment (ntn-x2) [inline docs/concepts/credentials/3_terms-and-quote.md]: We're using attester and claimer lowercase here. I guess we will have an editorial round afterwards once we define all the style guidelines.

File: docs/concepts/credentials/3_terms-and-quote.md [hunk 15]
```diff
@@ -78,30 +73,14 @@ The terms object is sent via the messaging in the SDK using the interface “ISu
 
 ## How to create a quote
 
-A quote is a description of the terms and conditions of the work to be performed. It may be sent to the attester from the claimer, but may also be sent from a claimer to an attester. In cases where multiple attesters provide the same attestation (for example, a car inspection) the claimer may send a quote to several attesters to choose the attester with the best conditions.
+A quote consists of costs, a timeframe and the terms and conditions of the work to be performed. It may be sent to the claimer by the attester. In cases where multiple attesters provide the same attestation (for example, a car inspection) the claimer may request a quote of several attesters to choose the attester with the best conditions.
```
Comment (ntn-x2) [inline docs/concepts/credentials/3_terms-and-quote.md]: Timeframe would be the expiration date?

File: docs/concepts/credentials/3_terms-and-quote.md [hunk 16]
```diff
@@ -140,60 +119,11 @@ const agreedSignedQuoteObj = {
...
 ```
 
-## How to create a request for attestation
-
-Once the terms have been agreed, the claimer must accept the conditions to move into the request for attestation. The claimer needs to submit a complete claim using the following process:
+## Agreeing to Terms and Quote
 
-- If the claimer agrees to the terms the claimer signs the `IRequestForAttestation` object.
-- If the attester sent a quote as part of the agreement, the claimer must also sign and agree to the attester's signed quote `IQuoteAttesterSigned`.
+If the claimer agrees to the `Terms`, he includes the information for it in the `Request for Attestation` object.
```
Comment (ntn-x2) [inline docs/concepts/credentials/3_terms-and-quote.md]:
```suggestion
If the claimer agrees to the `Terms`, they include the information for it in the `Request for Attestation` object.
```

**Review by ntn-x2 — CHANGES_REQUESTED** [review]
As I wasn't there during planning, I am not sure about what I am going to say, but it seems that there is just too much information in the concepts section. Not sure how much we could take out of here, but probably stuff like quotes, challenges, etc, should be moved to the advanced section.

File: docs/concepts/credentials/1_ctypes.md [hunk 17]
```diff
@@ -39,6 +39,21 @@ When making a claim for a CType, all the following properties are required:
...
+
+```js title="CType example"
+{
+  schema:  {...} // as defined in the example above>,
+  owner: null, // DID of the owner, or null
+  hash: '0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010' // For looking up on-chain
+}
+```
+
+## Nested CTypes
```
Comment (weichweich) [inline docs/concepts/credentials/1_ctypes.md]: After reading this section I'm not sure how this nesting works.
My expectation was that you just have another CType in your CType. So you would always have all properties and the full structure nested in the outer CType.
But the referencing looks different. You only nest fields of the CType in the new CType? Maybe this could be made clearer?

I would maybe also add a few diagrams that do some eli5.
Overall I would even go so far as to replace all json examples with a higher level abstraction like a box with a few fields in there.

File: docs/concepts/credentials/1_ctypes.md (hunk 5 above)
Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: good question. IDK, but I was thinking about it, too

File: docs/concepts/credentials/1_ctypes.md (hunk 17 above)
Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: ok, I will try to explain it better. And yes, the nesting happens by reference

File: docs/concepts/credentials/0_overview.md (hunk 1 above)
Comment (tjwelde) [reply docs/concepts/credentials/0_overview.md]: de5620fe

File: docs/concepts/credentials/0_overview.md (hunk 2 above)
Comment (tjwelde) [reply docs/concepts/credentials/0_overview.md]: de5620fe

File: docs/concepts/credentials/0_overview.md (hunk 3 above)
Comment (tjwelde) [reply docs/concepts/credentials/0_overview.md]: de5620fe

File: docs/concepts/credentials/0_overview.md (hunk 4 above)
Comment (tjwelde) [reply docs/concepts/credentials/0_overview.md]: de5620fe

File: docs/concepts/credentials/1_ctypes.md (hunk 6 above)
Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: I am not sure about capitalisation. I felt like every KILT Concept we have could be capitalised, because they act as proper nouns. But maybe we just align that in the end.

File: docs/concepts/credentials/2_claiming.md (hunk 11 above)
Comment (tjwelde) [reply docs/concepts/credentials/2_claiming.md]: I guess it is just here, in case the reader didn't read the previous section, But I can also remove it and just reference the ctype section, where needed.

File: docs/concepts/credentials/2_claiming.md (hunk 13 above)
Comment (tjwelde) [reply docs/concepts/credentials/2_claiming.md]: I think it can

File: docs/concepts/credentials/3_terms-and-quote.md (hunk 15 above)
Comment (tjwelde) [reply docs/concepts/credentials/3_terms-and-quote.md]: No, a timeframe for when the attestation will be created, AFAIK

Comment (tjwelde) [reply docs/concepts/credentials/3_terms-and-quote.md]: 317bd4334a1cbfdb4f1aa7b1bbd7b7e8b7f2dd71

File: docs/concepts/credentials/3_terms-and-quote.md (hunk 14 above)
Comment (tjwelde) [reply docs/concepts/credentials/3_terms-and-quote.md]: 317bd4334a1cbfdb4f1aa7b1bbd7b7e8b7f2dd71

File: docs/concepts/credentials/1_ctypes.md (hunk 6 above)
Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: a270146b

Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: a270146b

File: docs/concepts/credentials/1_ctypes.md (hunk 7 above)
Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: a270146b

File: docs/concepts/credentials/1_ctypes.md (hunk 8 above)
Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: a270146b

File: docs/concepts/credentials/2_claiming.md (hunk 10 above)
Comment (tjwelde) [reply docs/concepts/credentials/2_claiming.md]: a270146b

File: docs/concepts/credentials/2_claiming.md (hunk 12 above)
Comment (tjwelde) [reply docs/concepts/credentials/2_claiming.md]: a270146b

File: docs/concepts/credentials/2_claiming.md (hunk 13 above)
Comment (tjwelde) [reply docs/concepts/credentials/2_claiming.md]: a270146b

File: docs/concepts/credentials/3_terms-and-quote.md (hunk 16 above)
Comment (tjwelde) [reply docs/concepts/credentials/3_terms-and-quote.md]: a270146b

File: docs/concepts/credentials/3_terms-and-quote.md (hunk 15 above)
Comment (ntn-x2) [reply docs/concepts/credentials/3_terms-and-quote.md]: Can you maybe add an explanation of what timeframe is supposed to mean?

File: docs/concepts/02_Credentials/01_overview.md [hunk 18]
```diff
@@ -0,0 +1,24 @@
+---
+id: overview
+title: Overview
+---
+
+**Credentials** are attested claims which are owned by a **Claimer**, attested by an **Attester** and can be verified by a **Verifier**.
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/01_overview.md]: Should we perhaps include a diagram here of the different interactions? Or do we want to do it later?

File: docs/concepts/02_Credentials/02_ctypes.md [hunk 19]
```diff
@@ -0,0 +1,118 @@
...
+  - URI format e.g. https://www.example.com
+
+```js title="CType schema example"
+{
+  $id: 'kilt:ctype:0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010',
+  $schema: 'http://kilt-protocol.org/draft-01/ctype#',
+  title: 'CtypeModel 2',
+  properties: {
+    name: { type: 'string' },
+    age: { $ref: `kilt:ctype:0x1`, },
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/02_ctypes.md]: Only one hex character is not a valid byte representation

```suggestion
    age: { $ref: `kilt:ctype:0x01`, },
```

File: docs/concepts/02_Credentials/02_ctypes.md [hunk 20]
```diff
@@ -0,0 +1,118 @@
...
+    },
+  type: 'object',
+}
+```
+
+The CType schema is afterwards wrapped into the full CType object:
+
+```js title="CType example"
+{
+  schema:  {...} // as defined in the example above>,
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/02_ctypes.md]:
```suggestion
  schema:  {...} // as defined in the example above,
```

File: docs/concepts/02_Credentials/02_ctypes.md [hunk 21]
```diff
@@ -0,0 +1,118 @@
...
+
+### Constructing the `hash` for the `$id`
+
+KILT uses the hashing algorithm blake2b, hashing the CType contents. The object is sorted by a canonicalization algorithm before hashing to ensure that semantically equivalent CTypes with different order of their properties would produce the same output hash.
+
+The `hash` is made from the following fields in the schema object:
+
+- `$schema`
+- `properties`
+  - `$id`
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/02_ctypes.md]: Should we then remove this since it is not used in the hashing process?

File: docs/concepts/02_Credentials/03_claiming.md [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
+title: Claiming
+---
+As KILT is an open system, any entity can make a claim about themselves. A claim (as in the real world) can only be trusted, if another trusted entity (we call them Attesters) confirms this claim. A Verifier therefore trusts specific Attesters, who attest Claims from Claimers.
+
+In KILT, claims are based on claim types (CTypes). A claimer can either create a new CType or, when KILT is established and standard CTypes are available, may use an existing CType when creating their claim.
+
+## Create your Claim from a CType
+
+Once you have a CType, you only need to fill it with content to create your claim, i.e. supply the values for all fields of the CType.
+The resulting claim is referencing the ctype by its hash and includes the DID of the owner of the claim.
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/03_claiming.md]:
```suggestion
The resulting claim is referencing the ctype by its hash and includes the DID of the owner (i.e., the subject) of the claim.
```

Not sure this is the best way, but AFAIK there's currently no distinction between the owner and subject of a claim, and subject makes it easier to understand.

File: docs/concepts/02_Credentials/04_attestation.md [hunk 23]
```diff
@@ -0,0 +1,25 @@
...
+
+To create the attestation, the attester:
+
+- manually verifies the data inside the claim by checking that all the data fields match the requirements of the attestation
+- creates an attestation object from the root hash of the RequestForAttestation and the attester’s address.
+- stores the attestation hash on the chain by submitting a new transaction with the attester’s signature.
+
+Once this is done, the attester:
+
+- creates an `AttestedClaim` object from the `RequestForAttestation` and the attestation objects
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/04_attestation.md]:
```suggestion
- creates a `Credential` object from the `RequestForAttestation` and the attestation objects
```

File: docs/concepts/02_Credentials/04_attestation.md [hunk 24]
```diff
@@ -0,0 +1,25 @@
...
+- encrypts the `AttestedClaim` object
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/04_attestation.md]:
```suggestion
- encrypts the `Credential` object
```

File: docs/concepts/02_Credentials/04_attestation.md [hunk 25]
```diff
@@ -0,0 +1,25 @@
...
+- sends the encrypted `AttestedClaim` object back to the claimer.
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/04_attestation.md]:
```suggestion
- sends the encrypted `Credential` object back to the claimer.
```

File: docs/concepts/02_Credentials/04_attestation.md (hunk 24 above)
Comment (ntn-x2) [reply docs/concepts/02_Credentials/04_attestation.md]: In the same way we have recommended to use encryption for the claimer, we should do the same here. I.e., we should not include encryption as a step in the process, but should say to send the object back to the claimer (even thought this process will soon change). We could then mention that it is recommended to use the previously established encrypted channel, if one was created.

File: docs/concepts/02_Credentials/05_verification.md [hunk 26]
```diff
@@ -0,0 +1,50 @@
+---
+id: verification
+title: Verification
+---
+
+KILT allows a verifier to check if the information in an attested claim (credential) presented by a claimer is correct and valid as, in addition to the credential, the claimer also presents evidence to the validator that a third party (attester) ensured the correctness of the claimer’s attributes.
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/05_verification.md]: Don't we want to move away from the words "attested claim"?

File: docs/concepts/02_Credentials/05_verification.md [hunk 27]
```diff
@@ -0,0 +1,50 @@
...
+- The claimer needs their credential and the private key associated with their identifier.
+- The verifier needs the identifier of the trusted attester
+
+During the verification process the claimer wants to prove three things to the verifier:
+
+- The attestation for the claim is valid
+- The claim actually belongs to the claimer presenting it
+- The claim contains the information that the verifier is interested in
+
+## Presenting a Credential with Selective Disclosure
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/05_verification.md]: Wasn't this supposed to be part of the advanced topics?

File: docs/concepts/02_Credentials/05_verification.md [hunk 28]
```diff
@@ -0,0 +1,50 @@
...
+
+The claimer can decide how much of the information in their credential they wish to reveal before they send it to the verifier.
+They can choose to hide attributes and thus only disclose a subset of the data.
+For example, if sending a driving license to confirm their age, they may want to show only their name and date of birth, while withholding their address and any other details.
+This improves the privacy of the claimer since they only need to show attributes that are required in the specific context.
```
Comment (ntn-x2) [inline docs/concepts/02_Credentials/05_verification.md]: I would also mention that even though it increases privacy, it does not eliminate correlation between presentations, as the root hash of the credential is always the same and represents the main linking factor across presentations.

File: docs/concepts/05_Advanced/terms-and-quote.md [hunk 29]
```diff
@@ -0,0 +1,129 @@
...
+
+- **Quote**: Details on how to create a Quote are outlined here. In brief, the Quote could include any or all of the following:
+
+  - The time frame the claim will be completed in
+  - The public address of the Attester completing the attestation
+  - The hash of the corresponding CType
+  - A breakdown of all the (net and gross) costs, the currency quoted, and any taxes associated with the attestation.
+  - A link to the terms and conditions of the attestation.
+
+- **Delegations**: An Attester may include the right to attest to a specific claim from a top-down trust authority e.g. a business giving the right to their employee to attest the claim on their behalf. This is a delegation. If the Attester is delegating their attestation rights to another entity, this should be stated clearly at this point. See here for details on how to create a delegation.
```
Comment (ntn-x2) [inline docs/concepts/05_Advanced/terms-and-quote.md]: here where? 😆

File: docs/concepts/05_Advanced/terms-and-quote.md [hunk 30]
```diff
@@ -0,0 +1,129 @@
...
+
+## How to create a Terms object using the KILT SDK
```
Comment (ntn-x2) [inline docs/concepts/05_Advanced/terms-and-quote.md]: I feel we have crossed the line of what can be here and what could go in the cookbook. Does **Not even JSON** also applies to the advanced concepts? 😛

File: docs/develop/01_sdk/02_Cookbook/04_Issue and verify a KILT credential/01_ctypes.md [hunk 31]
```diff
@@ -16,7 +16,7 @@ KILT uses [JSON-Schema](https://json-schema.org/) (currently draft-07) to valida
 
 ### Properties
 
-When making a claim for a CType, all the following properties are required:
```
Comment (ntn-x2) [inline docs/develop/01_sdk/02_Cookbook/04_Issue and verify a KILT credential/01_ctypes.md]: This seems to be a repetition of what's in the concepts.

**Review by ntn-x2 — COMMENTED** [review]
I realized almost at the end that this PR is definitely still WIP, so please disregard comments about stuff that has not been updated yet. But I also hope some of the nitpicks are good suggestions 😁

File: docs/concepts/02_Credentials/05_verification.md (hunk 26 above)
Comment (tjwelde) [reply docs/concepts/02_Credentials/05_verification.md]: Yeah, most of this was just copied from the core-feature section, but still valid point

File: docs/concepts/02_Credentials/05_verification.md (hunk 27 above)
Comment (tjwelde) [reply docs/concepts/02_Credentials/05_verification.md]: Didn't get to it, yet

File: docs/develop/01_sdk/02_Cookbook/04_Issue and verify a KILT credential/01_ctypes.md (hunk 31 above)
Comment (tjwelde) [reply docs/develop/01_sdk/02_Cookbook/04_Issue and verify a KILT credential/01_ctypes.md]: hmm, interesting, these changes must've been taken over to these files (which are the original core-feature files, which were copied and adjusted for the concept section), while doing the big merge.

File: docs/concepts/05_Advanced/terms-and-quote.md (hunk 30 above)
Comment (tjwelde) [reply docs/concepts/05_Advanced/terms-and-quote.md]: Puh, yeah I am not so sure either.
Do we only show the function calls in the cookbook, or also the resulting JSON objects?

File: docs/concepts/02_Credentials/01_overview.md (hunk 18 above)
Comment (tjwelde) [reply docs/concepts/02_Credentials/01_overview.md]: 4b1d7d08a6aed1117c05c5a218f07962fd249cb1 like this?

File: docs/concepts/credentials/2_claiming.md (hunk 9 above)
Comment (tjwelde) [reply docs/concepts/credentials/2_claiming.md]: was put into the overview page

File: docs/concepts/02_Credentials/01_overview.md (hunk 18 above)
Comment (ntn-x2) [reply docs/concepts/02_Credentials/01_overview.md]: Like it! I would perhaps remove the black background from the dark mode image, and only leave the white text. So if we change the background colour of the dark mode, we don't have to change the background of those labels as well.

File: docs/concepts/05_Advanced/terms-and-quote.md (hunk 30 above)
Comment (ntn-x2) [reply docs/concepts/05_Advanced/terms-and-quote.md]: It's saying "using the KILT SDK", so for me it feels natural to put it in the cookbook. In the concepts (although an advanced one), I would focus on what terms and quotes are, and what they are used for.

File: docs/concepts/02_Credentials/01_overview.md (hunk 18 above)
Comment (tjwelde) [reply docs/concepts/02_Credentials/01_overview.md]: Problem is, that without the backgrounds the line will be too near of the text

File: docs/concepts/credentials/1_ctypes.md (hunk 5 above)
Comment (tjwelde) [reply docs/concepts/credentials/1_ctypes.md]: I removed most of them

File: docs/develop/01_sdk/02_Cookbook/04_Issue and verify a KILT credential/01_ctypes.md (hunk 31 above)
Comment (tjwelde) [reply docs/develop/01_sdk/02_Cookbook/04_Issue and verify a KILT credential/01_ctypes.md]: removed them

**Comment by tjwelde** [comment]
Ok, I think it is much simplified now. Sorry for not referencing a commit on every change request you guys did, but I had to change stuff all over the place. The examples are mostly removed, I still found them good in the ctype section, but that can be argued about. We definitely will need some JSON examples in the cookbook I think.

**Review by ntn-x2 — APPROVED** [review]
Veeeeeery nice! 👌

**Review by wischli — APPROVED** [review]
I suppose I should have reviewed this before #104. I made a bunch of newline, and capitalization inconsistencies remarks there. It does not make sense to replicate these here. Sorry about that @ntn-x2.

Content-wise, LGTM.
